src/tests/multihost/ad/test_memcache_sid.py

```diff
@@ -0,0 +1,77 @@
+""" Tests for Memcache for SIDs
+
+:requirement: Memory cache for SID
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+
+import os
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+
+
+@pytest.mark.usefixtures('joinad')
+@pytest.mark.tier1_3
+@pytest.mark.memcachesid
+class Testmemcachesid(object):
+    def test_0001_memcache_sid(self, multihost):
+        """
+        :title: Verify memcache for SID
+        :id: f7fce9c5-5ba6-428b-8e9b-5e07a88b5050
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1245367
+        :customerscenario: true
+        :steps:
+          1. get uid of user
+          2. get gid of group
+          3. Clear sssd cache
+          4. Lookup sid-by-uid with pysss_nss_idmap
+          5. Lookup sid-by-uid with pysss_nss_idmap again
+          6. Lookup sid-by-gid with pysss_nss_idmap
+          7. Lookup sid-by-gid with pysss_nss_idmap again
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Lookup should run through nss responder
+          5. Lookup should read from memory cache of SID
+          6. Lookup should run through nss responder
+          7. Lookup should read from memory cache of SID
+        """
+        req_pkg = 'yum install -y strace python3-libsss_nss_idmap'
+        multihost.client[0].run_command(req_pkg, raiseonerr=True)
+        ad_domain = multihost.ad[0].domainname
+        ad_user = f'user2@{ad_domain}'
+        ad_group = f'group2@{ad_domain}'
+        lookup_cmd_user = f'id -u {ad_user}'
+        cmd = multihost.client[0].run_command(lookup_cmd_user, raiseonerr=True)
```

Comment:
raiseonerr always True , need to mention it if False

Reply:
done